server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/mail/EmailQuery.scala

```diff
@@ -23,7 +23,7 @@ import com.google.common.hash.Hashing
 import org.apache.james.jmap.model.AccountId
 import org.apache.james.mailbox.model.{MailboxId, MessageId}
 
-case class EmailQueryRequest(accountId: AccountId, inMailbox: Option[MailboxId])
+case class EmailQueryRequest(accountId: AccountId, inMailbox: Option[MailboxId], inMailboxOtherThan: Option[Seq[MailboxId]])
```

Comment:
I proprose we add a EmailQueryFilter case class holding the inMailbox and inMailboxOtherThan properties